capa/features/extractors/viv/syscall.py

```diff
@@ -0,0 +1,1359 @@
+from typing import Callable
+
+import viv_utils
+import viv_utils.flirt
+import envi.archs.arm.disasm
+import envi.archs.i386.disasm
+import envi.archs.amd64.disasm
+
+from capa.features.common import ARCH_ARM, ARCH_I386, ARCH_AMD64
+from capa.features.address import AbsoluteVirtualAddress
+from capa.features.extractors.viv.helpers import read_memory
+from capa.features.extractors.base_extractor import BBHandle, FeatureExtractor
+
+
+def getSyscallName(num: int, arch: bool) -> str:
```

Comment:
```suggestion
def get_syscall_name(num: int, arch: bool) -> str:
```

and maybe document this can raise `KeyError` when not found ?